detectors/node/opentelemetry-resource-detector-aws/test/detectors/AwsBeanstalkDetector.test.ts

```diff
@@ -16,7 +16,7 @@
 
 import * as assert from 'assert';
 import * as sinon from 'sinon';
-import { awsBeanstalkDetector, AwsBeanstalkDetector } from '../../src';
+import { awsBeanstalkDetector, AwsBeanstalkDetectorSync } from '../../src';
```

Comment:
note for reviewer: the `AwsBeanstalkDetector` became a wrapper for `AwsBeanstalkDetectorSync` class so we have to stub the later, hence the import and the changes in stubs